src/screens/MemberScreen/MembersPage.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  StyleSheet,
+  Text,
+  View,
+  FlatList,
+  ActivityIndicator,
+} from 'react-native';
+import React, { useEffect, useState } from 'react';
+import SearchBar from '../../components/SearchBar';
+import RenderMemberItem from '../../components/ToDoComponent/RenderMemberItem';
+import { useNavigation, RouteProp, useRoute } from '@react-navigation/native';
+
+type MembersPageRouteProp = RouteProp<RootStackParamList, "Member's page">;
+
+const MembersPage = () => {
+  const route = useRoute<MembersPageRouteProp>();
+  const [membersData, setMembersData] = useState([]);
+  const [filterMemberData,setFilterMemberData] = useState([])
+  const { selectedMember, setSelectedMember } = route.params;
+  const [searchValue, setSearchValue] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<null | string>(null);
+
+  useEffect(() => {
+    callMembersApi();
+  }, [selectedMember]);
+
+  const callMembersApi = async () => {
+    try {
+      // Set loading state
+      setLoading(true);
+
+      const members = await fetch('https://api.realdevsquad.com/members');
```

Comment:
Please the constants for the API URL and fetch is default or we defined ?

Reply:
moved to constant folder